src/PrivateDns/PrivateDns/Zones/SetAzurePrivateDnsZone.cs

```diff
@@ -0,0 +1,104 @@
+﻿// ------------------------------------------------------------------------------------------------
+// <copyright file="SetAzurePrivateDnsZone.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.PrivateDns.Zones
+{
+    using System.Collections;
+    using System.Management.Automation;
+    using Microsoft.Azure.Commands.PrivateDns.Models;
+    using Microsoft.Azure.Commands.PrivateDns.Utilities;
+    using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+    using ProjectResources = Microsoft.Azure.Commands.PrivateDns.Properties.Resources;
+
+    /// <summary>
+    /// Updates an existing zone.
+    /// </summary>
+    [Cmdlet("Set", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "PrivateDnsZone", SupportsShouldProcess = true, ConfirmImpact = ConfirmImpact.Medium), OutputType(typeof(PrivateDnsZone))]
+    public class SetAzurePrivateDnsZone : PrivateDnsBaseCmdlet
```

Comment:
What is the default param set of this cmdlet?

Reply:
added it.